### DISCUSSION TITLE
Partick Collins Once Said ?

### DISCUSSION THREAD

**Discussion by Blockchain-Oracle** [Cyfrin/foundry-full-course-cu#1353] [General]
### We are Engineers. We work incredibly hard to be incredibly lazy. 🙃
@PatrickAlphaC

**Comment by PatrickAlphaC** [comment]
I will work day and night if it means I have to press one less button

**Comment by 0xmillysmith** [comment]
**"one Less Button", Elaborate sir.**

Let's Get Froggy 🐸

**Reply by PatrickAlphaC** [reply]
A key stroke on the keyboard.
A mouse click.

Anything.

The more buttons your press, the worse.
